### DISCUSSION TITLE
카테고리 펼치기 기능 제공되나요?

### DISCUSSION THREAD

**Discussion by zineeworld** [pronist/hELLO#260] [Q&A] [answered]
안녕하세요 티스토리 스킨 너무 감사하게 잘 쓰고 있습니다.

2022년에 이런 글이 있는 걸 찾아보긴 했는데
이후에 업데이트 반영이 된 건가 문의드립니다.

https://github.com/pronist/hello/discussions/131#discussion-4430482

지금은 기본값이 접힌 상태인 것 같은데 어떻게 펼침 상태로 해둘 수 있는지 모르겠네요 ㅜㅜ

**Comment by pronist** [comment, marked as answer]
@zineeworld 스킨 옵션에서 카테고리 접기/펼치기 부분 살펴보시면 됩니다.

**Comment by zineeworld** [comment]
감사합니다!
